src/libMessage/Messenger.cpp

```diff
@@ -3885,6 +3962,19 @@ bool Messenger::SetDSPoWSubmission(
     return false;
   }
 
+  // Governance data handling. first=proposalId,second=votevalue
+  if (govProposal.first > 0 && govProposal.second > 0) {
+    result.mutable_data()->mutable_govdata()->set_proposalid(govProposal.first);
+    result.mutable_data()->mutable_govdata()->set_votevalue(govProposal.second);
+    LOG_GENERAL(INFO, "Governance: SetVote in pow submission: GovProposalId:"
+                          << govProposal.first
+                          << " GovVoteValue:" << govProposal.second);
+    if (!result.data().govdata().IsInitialized()) {
+      LOG_GENERAL(WARNING,
+                  "DSPoWSubmission governance data initialization failed");
+    }
```

Comment:
Do you want to return false here? Just checking

Reply:
Intentionally not returning from here.Otherwise it will impact normal behavior if theirs some problem in governance message serialization.